src/main/java/uk/gov/hmcts/reform/cpo/security/AuthError.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cpo.security;
+
+/**
+ * Authentication and Authorisation error messages.
+ **/
```

Comment:
We can use lombok .  
I am not sure but lombok could gives us some facilities. for final classes with private constructors
@value(staticConstructor = "getInstance") for custom creation or others

Reply:
No adjustment made.  This class holds constatns only.  I reviewed switching to an enum but as this method of defining constants in classes has been used throughout CCD I have decided not to make the change.